File: week_10/c_sharp_course/week_1/day_5/try_catch/code/start_code/ExampleTryCatch.cs
using System;

public class ExampleTryCatch
{
  PetShop shop;

  public void Run()
  {
    this.Setup();
    Pet found = shop.FindPetByName("Meowingtons");
    Console.WriteLine("Found pet: " + found.GetName());
  }

  public void Setup()
  {
    shop = new PetShop();
    shop.AddPet(new Dog("Rover"));
    shop.AddPet(new Cat("MEOWingtons"));
    shop.AddPet(new Fish("FINlay"));
  }
}

File: week_07/day_1/Try_Catch_Exceptions/code/Try_Catch_Start/app/src/main/java/example/codeclan/com/try_catch_start/ExampleTryCatch.java
package example.codeclan.com.try_catch_start;

/**
 * Created by user on 30/08/2017.
 */

public class ExampleTryCatch{
    PetShop shop;

    public void run(){
        setup();
        Pet found = shop.findPetByName("Meowingtons");
        System.out.println("Found pet: " + found.getName());
    }

    public void setup(){
        shop = new PetShop();
        shop.addPet(new Dog("Rover"));
        shop.addPet(new Cat("MEOWingtons"));
        shop.addPet(new Fish("FINlay"));
    }
}

